### ISSUE TITLE
AppArgs - need to replace slashes \ into /

### ISSUE THREAD

**Issue by engor** [blitz-research/monkey2#111]
FilePaths from AppArgs have slashes like C:\folder1\folder2\ .
But functions like RealPath working with inverse slashes like C:/folder1/folder2/.
So we got incorrect RealPath for absolute path from AppArgs.

**Comment by blitz-research** [comment]
Yeah, not sure what to do here as AppArgs aren't necessarily filepaths. I
really need to fix '\' in filesystem I think.

On Sat, Nov 19, 2016 at 7:25 AM, Evgeniy Goroshkin <notifications@github.com
